### DISCUSSION TITLE
Galaxy Brain 획득 요청 방법

### DISCUSSION THREAD

**Discussion by learningssik** [learningssik/pull-request#123] [Announcements]
안녕하세요. 제 Discussion 에 많은 분들이 방문해서 Galaxy Brain 뱃지를 위한 채택을 요청하셨습니다.
다만 이력성으로 되어 있어 제가 일일히 확인하지 못하고 넘어 가는 경우가 많아 공지글을 작성하게 되었습니다.

Galxy Barin 요청 순서
1. 현재 게시글에 답변으로 뱃지 요청 남기기
2. 제가 확인 후 discussion 2를 생성하고 대댓글로 링크를 올리도록 하겠습니다.
3. 해당 링크에 들어가서 글 남겨주시면 제가 채택해드리도록 하겠습니다.

2년이 지났지만 종종 방문해주셔서 감사하고, 도움이 되셨다면 해당 레포 우측 상단에 Star(⭐) 클릭해주시면 감사하겠습니다.

여러 일들 때문에 많이 신경을 쓰지 못하였지만 혼자서 정리한 유용한 내용들을 앞으로 더 공유할 수 있도록 하겠습니다.

**Comment by falconlee236** [comment]
안녕하세요 블로그 보고 벳지 신청하려고 왔습니다! @learningssik

**Reply by learningssik** [reply]
안녕하세요!
#124 #125 에 답변다시면 채택드리겠습니다!
늦어서 죄송합니다 ㅠㅠ

**Reply by falconlee236** [reply]
답변 달았는데 채택이 늦네요 ㅠㅠㅠ

**Reply by learningssik** [reply]
늦어서 죄송합니다..! 확인해보세요.

**Reply by falconlee236** [reply]
뱃지 무사히 받았습니다 감사합니다!

**Comment by hyeonsang010716** [comment]
안녕하세요~! 블로그 보고 벳지 신청하려고 왔습니다 :) @learningssik

**Reply by learningssik** [reply]
안녕하세요. #126 #127 에 댓글 남겨주세요!
